docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -286,3 +286,92 @@ POST /sales/_search?size=0
 // TEST[setup:sales]
 
 <1> Documents without a value in the `quantity` field will fall into the same bucket as documents that have the value `0`.
+
+[[search-aggregations-bucket-histogram-aggregation-histogram-fields]]
+==== Histogram fields
+
+Running a histogram aggregation over histogram fields computes the total number of counts for each interval.
+
+For example, for the following index that stores pre-aggregated histograms with latency metrics (in milliseconds) for different networks:
+
+[source,console]
+--------------------------------------------------
+PUT metrics_index/_doc/1
+{
+  "network.name" : "net-1",
+  "latency_histo" : {
+      "values" : [1, 3, 8, 12, 15],
+      "counts" : [3, 7, 23, 12, 6]
+   }
+}
+
+PUT metrics_index/_doc/2
+{
+  "network.name" : "net-2",
+  "latency_histo" : {
+      "values" : [1, 6, 8, 12, 14],
+      "counts" : [8, 17, 8, 7, 6]
+   }
+}
+
+POST /metrics_index/_search?size=0
+{
+    "aggs" : {
+        "latency_buckets" : {
+            "histogram" : {
+                "field" : "latency_histo",
+                "interval" : 5
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+
+The `histogram` aggregation will sum the counts of each interval computed based on the `values` and
+return the following output:
+
+[source,console-result]
+--------------------------------------------------
+{
+    ...
+    "aggregations": {
+        "prices" : {
+            "buckets": [
+                {
+                    "key": 0.0,
+                    "doc_count": 18
+                },
+                {
+                    "key": 5.0,
+                    "doc_count": 48
+                },
+                {
+                    "key": 10.0,
+                    "doc_count": 25
+                },
+                {
+                    "key": 15.0,
+                    "doc_count": 6
+                }
+            ]
+        }
+    }
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:test not setup]
+
+[IMPORTANT]
+========
+Histogram aggregation is a bucket aggregation, which partitions documents into buckets rather than calculating metrics over fields like
+metrics aggregations do. Each bucket represents a collection of documents which sub-aggregations can run on.
+On the other hand, a histogram field is a pre-aggregated field representing multiple values inside a single field:
+buckets of numerical data and a count of items/documents for each bucket. This mismatch between the histogram aggregations expected input
+(expecting raw documents) and the histogram field (that provides summary information) limits the outcome of the aggregation
+to only the doc counts for each bucket.
+
+
+**Consequently, when executing a histogram aggregation over a histogram field, no sub-aggregations are supported.**
+========
```

Comment:
Perhaps we need to improve wording here. This is a subtle concept and we must be as clear as possible on the message